game/src/main/java/org/apollo/game/release/r181/decoders/EventKeyboardDecoder.java

```diff
@@ -0,0 +1,29 @@
+package org.apollo.game.release.r181.decoders;
+
+import org.apollo.game.message.impl.decode.EventKeyboardMessage;
+import org.apollo.net.codec.game.DataTransformation;
+import org.apollo.net.codec.game.DataType;
+import org.apollo.net.codec.game.GamePacket;
+import org.apollo.net.codec.game.GamePacketReader;
+import org.apollo.net.release.MessageDecoder;
+
+import java.time.Duration;
+
+/**
+ * @author Khaled Abdeljaber
+ */
+public class EventKeyboardDecoder extends MessageDecoder<EventKeyboardMessage> {
```

Comment:
The name of this class is back-to-front. Should be `KeyboardEventDecoder`, similarly for the message (and `EventMouseMove`).